src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import duke.tasks.*;
+
+/** Loads and saves tasks from file*/
+public class Storage {
+    /** User's root directory path*/
+    private static final String userDir = System.getProperty("user.dir");
+    /** Data directory path*/
+    private static final Path dataDirPath = Paths.get(userDir, "data");
+    /** duke.txt file path*/
+    private static final Path dataFilePath = Paths.get(userDir, "data", "duke.Duke.txt");
```

Comment:
I think that variables declared as final should be in capital letters so that its clearer to read and understand, when referenced to later in the code